backend/python_backend/model_routes.py

```
@@ -9,52 +9,55 @@
     model_name: str,
+    current_user: str = Depends(get_current_active_user),
     model_manager: ModelManager = Depends(get_model_manager),
 ):
-    """Loads a specific model into memory."""
+    """Loads a specific model into memory.
+    
+    Requires authentication.
+    """
     try:
         model_manager.load_model(model_name)
         return {"message": f"Model '{model_name}' loaded successfully."}
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)